src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,171 @@
+import { useState } from 'react';
+import { AiOutlineGlobal } from 'react-icons/ai';
+import LanguageDropdown from './LanguageDropdown';
```

Comment:
Please leave a line between global and local imports.

Reply:
As I mentioned at the meeting, the hamburger menu continues the previous action.
If the hamburger menu was left open, it will show open when it comes back. If the hamburger menu is left closed, it will show off when it comes back.
-I'll implement the other arrangement now.